### DISCUSSION TITLE
Solve to start without admin privileges in Win10/11

### DISCUSSION THREAD

**Discussion by ligonap** [zhongyang219/TrafficMonitor#1650] [General]
TrafficMonitor requires admin privileges all the time, so it is poorly programmed.
For normal users without admin privileges the following works:

Create a batch file _TrafficMonitor.bat_ with the content
```
  set __COMPAT_LAYER=RunAsInvoker
  start TrafficMonitor.exe
```
and copy it to the _TrafficMonitor.exe_ directory  ("_C:\Program Files\TrafficMonitor_") .

Run _regedit.exe_ in _C:\Windows_. Go to "_Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run_".
and create a new key "_TrafficMonitor_" with path "_C:\Program Files\TrafficMonitor\TrafficMonitor.bat_" to batch file.

Add the path "_C:\Program Files\TrafficMonitor_" in the environment variables.

After restart TrafficMonitor should start with the small main window and can be modified.

**Comment by OneTimePads** [comment]
No need for 

> Add the path "C:\Program Files\TrafficMonitor" in the environment variables.

in my case. Also directory can by under local user directory. "Program Files" require more privileges. Also 2 - CPU temp is missing, but GPU temp is there.
